packages/core/src/components/forms/numericInput.tsx

```diff
@@ -167,11 +167,15 @@ export class NumericInput extends AbstractPureComponent<HTMLInputProps & INumeri
      */
     private static FLOATING_POINT_NUMBER_CHARACTER_REGEX = /^[Ee0-9\+\-\.]$/;
 
+    private static CONTINUOUS_CHANGE_DELAY = 300;
+    private static CONTINUOUS_CHANGE_INTERVAL = 50;
```

Comment:
can we slow this down? how about 100.

Reply:
Hm, that feels almost too slow to me. I tried matching the native HTML input ([example](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number), I used Firefox for reference), but I realize now that the exact behavior of that might also be OS-dependent. Of course, since `NumericInput` is not a native element, there is no obligation to imitate it exactly. That being the case, maybe we could go all-out with a gradual ramp up in speed? :-)